packages/kbn-discover-utils/src/utils/nested_fields.test.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import type { DataView } from '@kbn/data-views-plugin/common';
+import { isNestedFieldParent } from './nested_fields';
+describe('isNestedFieldParent', () => {
+  it('correctly identifies nested parent fields', () => {
+    const nestedField = {
+      name: 'nested.field',
+      type: 'keyword',
+      subType: {
+        nested: {
+          path: 'nested.field.path',
+        },
+      },
+    };
+    const unnestedField = {
```

Comment:
It would be good to test with a field name that includes a `.` but isn't nested.

Reply:
good point, change applied 👍